<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:p="urn:schemas-microsoft-com:office:powerpoint"
xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta http-equiv=Content-Type content="text/html; charset=shift_jis">
<meta name=ProgId content=PowerPoint.Slide>
<meta name=Generator content="Microsoft PowerPoint 9">
<link id=Main-File rel=Main-File href="../3-3-2-4-4.htm">
<link rel=Preview href=preview.wmf>
<!--[if !mso]>
<style>
v\:* {behavior:url(#default#VML);}
o\:* {behavior:url(#default#VML);}
p\:* {behavior:url(#default#VML);}
.shape {behavior:url(#default#VML);}
v\:textbox {display:none;}
</style>
<![endif]-->
<title>タイトル</title>
<meta name=Description content="01/11/07: Discussion">
<link rel=Stylesheet href="master03_stylesheet.css">
<![if !ppt]>
<style media=print>
<!--.sld
	{left:0px !important;
	width:6.0in !important;
	height:4.42in !important;
	font-size:107% !important;}
-->
</style>
<script src=script.js></script><script><!--
gId="slide0016.htm"
if( !IsNts() ) Redirect( "PPTSld", gId );
//-->
</script><!--[if vml]><script>g_vml = 1;
</script><![endif]--><script for=window event=onload><!--
if( !IsSldOrNts() ) return;
if( MakeNotesVis() ) return;
LoadSld( gId );
MakeSldVis(0);
//-->
</script><![endif]><o:shapelayout v:ext="edit">
 <o:idmap v:ext="edit" data="18"/>
</o:shapelayout>
</head>

<body lang=JA style='margin:0px;background-color:black'
onclick="DocumentOnClick()" onresize="_RSW()" onkeypress="_KPH()">

<div id=SlideObj class=sld style='position:absolute;top:0px;left:0px;
width:534px;height:393px;font-size:16px;background-color:white;clip:rect(0%, 101%, 101%, 0%);
visibility:hidden'><p:slide coordsize="855,630"
 colors="#FFFFFF,#000000,#808080,#000000,#00CC99,#3333CC,#CCCCFF,#B2B2B2"
 masterhref="master03.xml">
 <p:shaperange href="master03.xml#_x0000_s1025"/><![if !ppt]><p:shaperange
  href="master03.xml#_x0000_s1028"/><![endif]><p:shaperange
  href="master03.xml#_x0000_m1026"/><v:shape id="_x0000_s18434" type="#_x0000_m1026"
  style='position:absolute;left:64.125pt;top:81.875pt;width:726.75pt;height:63pt'>
  <v:fill o:detectmouseclick="f"/>
  <v:stroke o:forcedash="f"/>
  <o:lock v:ext="edit" text="f"/>
  <p:placeholder type="title"/></v:shape><p:shaperange
  href="master03.xml#_x0000_m1027"/><v:shape id="_x0000_s18435" type="#_x0000_m1027"
  style='position:absolute;left:1in;top:174.25pt;width:714pt;height:371.75pt'>
  <v:fill o:detectmouseclick="f"/>
  <v:stroke o:forcedash="f"/>
  <o:lock v:ext="edit" text="f"/>
  <p:placeholder type="body" position="1"/></v:shape>
 <div v:shape="_x0000_s18434" class=T style='position:absolute;top:13.74%;
 left:8.42%;width:83.14%;height:9.16%;tab-interval:1.1545in'><span lang=EN-US
 style='font-family:"Times New Roman";mso-ascii-font-family:"Times New Roman";
 mso-fareast-language:JA'>Discussion</span></div>
 <div v:shape="_x0000_s18435" style='tab-interval:1.1545in'>
 <div class=B1 style='mso-margin-left-alt:557;mso-text-indent-alt:349'></div>
 <div class=B2 style='mso-margin-left-alt:861;mso-text-indent-alt:695'></div>
 <div class=B3 style='mso-margin-left-alt:1166;mso-text-indent-alt:1000'></div>
 <div class=B style='mso-margin-left-alt:0'><span style='position:absolute;
 top:28.49%;left:9.36%;width:91.57%'><span style='mso-special-format:nobullet;
 display:none'>&#8226;</span><span lang=EN-US style='font-family:"Times New Roman";
 mso-ascii-font-family:"Times New Roman";font-size:49%;mso-fareast-language:
 JA'>The two analytical methods were applied to simulated genotype data and
 real data, both of </span></span><span style='position:absolute;top:32.06%;
 left:9.36%;width:94.75%'><span lang=EN-US style='font-family:"Times New Roman";
 mso-ascii-font-family:"Times New Roman";font-size:49%;mso-fareast-language:
 JA'>which represented subtly heterogeneous two sub-populations. Result of
 analyses of simulated </span></span><span style='position:absolute;top:35.36%;
 left:9.36%;width:92.5%'><span lang=EN-US style='font-family:"Times New Roman";
 mso-ascii-font-family:"Times New Roman";font-size:49%;mso-fareast-language:
 JA'>data revealed both analytical methods could dissect two populations when
 SNPs with allele </span></span><span style='position:absolute;top:38.93%;
 left:9.36%;width:96.81%'><span lang=EN-US style='font-family:"Times New Roman";
 mso-ascii-font-family:"Times New Roman";font-size:49%;mso-fareast-language:
 JA'>frequency difference between populations more than 0.1 existed. Analysis 1
 (sum-of-chi square </span></span><span style='position:absolute;top:42.23%;
 left:9.36%;width:97.94%'><span lang=EN-US style='font-family:"Times New Roman";
 mso-ascii-font-family:"Times New Roman";font-size:49%;mso-fareast-language:
 JA'>values) showed statistical significance, even when SNPs with allele
 frequency difference of 0.05 </span></span><span style='position:absolute;
 top:45.8%;left:9.36%;width:96.06%'><span lang=EN-US style='font-family:"Times New Roman";
 mso-ascii-font-family:"Times New Roman";font-size:49%;mso-fareast-language:
 JA'>dominated, if such SNPs existed frequently, although analysis 2 (MCMC) did
 not show visible </span></span><span style='position:absolute;top:49.1%;
 left:9.36%;width:96.06%'><span lang=EN-US style='font-family:"Times New Roman";
 mso-ascii-font-family:"Times New Roman";font-size:49%;mso-fareast-language:
 JA'>discrimination in the distributions. In a situation, where
 population-based sampling association </span></span><span style='position:
 absolute;top:52.67%;left:9.36%;width:93.25%'><span lang=EN-US
 style='font-family:"Times New Roman";mso-ascii-font-family:"Times New Roman";
 font-size:49%;mso-fareast-language:JA'>studies were carried out, difference of
 allele frequencies among subtly stratified populations </span></span><span
 style='position:absolute;top:55.97%;left:9.36%;width:94.56%'><span lang=EN-US
 style='font-family:"Times New Roman";mso-ascii-font-family:"Times New Roman";
 font-size:49%;mso-fareast-language:JA'>would be negligible, as far as the
 difference between ethnic sub-populations was smaller than </span></span><span
 style='position:absolute;top:59.54%;left:9.36%;width:94.0%'><span lang=EN-US
 style='font-family:"Times New Roman";mso-ascii-font-family:"Times New Roman";
 font-size:49%;mso-fareast-language:JA'>the difference between phenotypic
 sub-groups. The comparison between results of simulated </span></span><span
 style='position:absolute;top:62.84%;left:9.36%;width:88.76%'><span lang=EN-US
 style='font-family:"Times New Roman";mso-ascii-font-family:"Times New Roman";
 font-size:49%;mso-fareast-language:JA'>data and real data gave an idea that
 the genetic structure between Tokyo and Osaka was </span></span><span
 style='position:absolute;top:66.41%;left:9.36%;width:94.94%'><span lang=EN-US
 style='font-family:"Times New Roman";mso-ascii-font-family:"Times New Roman";
 font-size:49%;mso-fareast-language:JA'>somewhere between a simulated data with
 maximum allele frequency difference of 0.025 and </span></span><span
 style='position:absolute;top:69.72%;left:9.36%;width:96.62%'><span lang=EN-US
 style='font-family:"Times New Roman";mso-ascii-font-family:"Times New Roman";
 font-size:49%;mso-fareast-language:JA'>another set with maximum allele
 frequency difference of 0.05. When complex genetic traits are </span></span><span
 style='position:absolute;top:73.02%;left:9.36%;width:96.44%'><span lang=EN-US
 style='font-family:"Times New Roman";mso-ascii-font-family:"Times New Roman";
 font-size:49%;mso-fareast-language:JA'>analyzed with LD mapping, allele
 frequency difference between phenotypically distinct groups </span></span><span
 style='position:absolute;top:76.59%;left:9.36%;width:85.2%'><span lang=EN-US
 style='font-family:"Times New Roman";mso-ascii-font-family:"Times New Roman";
 font-size:49%;mso-fareast-language:JA'>distributes most likely more than 0.05.
 From this standpoint, samples from Japanese </span></span><span
 style='position:absolute;top:79.89%;left:9.36%;width:81.64%'><span lang=EN-US
 style='font-family:"Times New Roman";mso-ascii-font-family:"Times New Roman";
 font-size:49%;mso-fareast-language:JA'>metropolises should be treated as
 homogeneous.</span></span></div>
 </div>
 <v:rect id="_x0000_s18436" style='position:absolute;left:44.875pt;top:48pt;
  width:765.125pt;height:524.25pt;mso-wrap-style:none;v-text-anchor:middle'
  filled="f" fillcolor="#0c9 [4]" strokecolor="silver" strokeweight=".5pt">
  <v:fill color2="white [0]"/>
  <v:stroke dashstyle="1 1"/>
  <v:shadow color="gray [2]"/>
 </v:rect><![if !vml]><img border=0 v:shapes="_x0000_s18436"
 src="slide0016_image060.gif" style='position:absolute;top:7.63%;left:5.24%;
 width:89.88%;height:83.71%'><![endif]></p:slide></div>

</body>

</html>
