<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:p="urn:schemas-microsoft-com:office:powerpoint"
xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta http-equiv=Content-Type content="text/html; charset=shift_jis">
<meta name=ProgId content=PowerPoint.Slide>
<meta name=Generator content="Microsoft PowerPoint 9">
<link id=Main-File rel=Main-File href="../3-3-2-4-4.htm">
<link rel=Preview href=preview.wmf>
<!--[if !mso]>
<style>
v\:* {behavior:url(#default#VML);}
o\:* {behavior:url(#default#VML);}
p\:* {behavior:url(#default#VML);}
.shape {behavior:url(#default#VML);}
v\:textbox {display:none;}
</style>
<![endif]-->
<title>タイトル</title>
<meta name=Description
content="01/11/07: Methods Genotyping Assay  Construction of genotype data for the structural analysis of real data">
<link rel=Stylesheet href="master03_stylesheet.css">
<![if !ppt]>
<style media=print>
<!--.sld
	{left:0px !important;
	width:6.0in !important;
	height:4.42in !important;
	font-size:107% !important;}
-->
</style>
<script src=script.js></script><script><!--
gId="slide0007.htm"
if( !IsNts() ) Redirect( "PPTSld", gId );
//-->
</script><!--[if vml]><script>g_vml = 1;
</script><![endif]--><script for=window event=onload><!--
if( !IsSldOrNts() ) return;
if( MakeNotesVis() ) return;
LoadSld( gId );
MakeSldVis(0);
//-->
</script><![endif]><o:shapelayout v:ext="edit">
 <o:idmap v:ext="edit" data="9"/>
</o:shapelayout>
</head>

<body lang=JA style='margin:0px;background-color:black'
onclick="DocumentOnClick()" onresize="_RSW()" onkeypress="_KPH()">

<div id=SlideObj class=sld style='position:absolute;top:0px;left:0px;
width:534px;height:393px;font-size:16px;background-color:white;clip:rect(0%, 101%, 101%, 0%);
visibility:hidden'><p:slide coordsize="855,630"
 colors="#FFFFFF,#000000,#808080,#000000,#00CC99,#3333CC,#CCCCFF,#B2B2B2"
 masterhref="master03.xml">
 <p:shaperange href="master03.xml#_x0000_s1025"/><![if !ppt]><p:shaperange
  href="master03.xml#_x0000_s1028"/><![endif]><p:shaperange
  href="master03.xml#_x0000_m1026"/><v:shape id="_x0000_s9218" type="#_x0000_m1026"
  style='position:absolute;left:64.125pt;top:81.875pt;width:726.75pt;height:94.5pt'>
  <v:fill o:detectmouseclick="f"/>
  <v:stroke o:forcedash="f"/>
  <o:lock v:ext="edit" text="f"/>
  <p:placeholder type="title"/></v:shape><p:shaperange
  href="master03.xml#_x0000_m1027"/><v:shape id="_x0000_s9219" type="#_x0000_m1027"
  style='position:absolute;left:48pt;top:224pt;width:756pt;height:350pt'>
  <v:fill o:detectmouseclick="f"/>
  <v:stroke o:forcedash="f"/>
  <o:lock v:ext="edit" text="f"/>
  <p:placeholder type="body" position="1"/></v:shape>
 <div v:shape="_x0000_s9218" class=T style='tab-interval:1.1545in'><span
 style='position:absolute;top:13.48%;left:8.42%;width:83.14%'><span lang=EN-US
 style='font-family:"Times New Roman";mso-ascii-font-family:"Times New Roman";
 font-size:41%;mso-fareast-language:JA'><i>Methods</i></span><span lang=EN-US
 style='mso-fareast-language:JA'><br>
 </span></span><span style='position:
 absolute;top:17.81%;left:8.42%;width:83.14%'><span lang=EN-US
 style='font-family:"Times New Roman";mso-ascii-font-family:"Times New Roman";
 font-size:55%;mso-fareast-language:JA'>Genotyping Assay <br>
 </span></span><span
 style='position:absolute;top:22.9%;left:.37%;width:99.43%'><span lang=EN-US
 style='font-family:"Times New Roman";mso-ascii-font-family:"Times New Roman";
 font-size:51%;mso-fareast-language:JA'>Construction of genotype data for the
 structural analysis of real data</span></span></div>
 <div v:shape="_x0000_s9219" style='tab-interval:1.1545in'>
 <div class=B1 style='mso-line-spacing:"90 20 0";position:absolute;top:36.13%;
 left:11.42%;width:81.83%;height:3.81%'><span style='position:absolute;
 top:0%;left:3.66%;width:96.1%'><span style='font-size:66%'><span
 style='mso-special-format:bullet;position:absolute;left:-3.8%'>&#8211;</span></span><span
 lang=EN-US style='font-family:"Times New Roman";mso-ascii-font-family:"Times New Roman";
 font-size:66%;mso-fareast-language:JA'>Invader assay with multiplex polymerase
 chain reaction (PCR), &#13;</span></span></div>
 <div class=B1 style='mso-line-spacing:"90 20 0";position:absolute;top:40.45%;
 left:11.42%;width:81.83%;height:3.81%'><span style='mso-special-format:nobullet;
 display:none'>&#8211;</span><span lang=EN-US style='font-family:"Times New Roman";
 mso-ascii-font-family:"Times New Roman";font-size:66%;mso-fareast-language:
 JA'><span style='mso-tab-count:1;width:3.69%'> </span>characterized as
 below:&#13;</span></div>
 <div class=B2 style='mso-line-spacing:"90 20 0";position:absolute;top:44.78%;
 left:16.29%;width:79.21%;height:3.3%'><span style='position:absolute;
 top:0%;left:3.07%;width:96.92%'><span style='font-size:64%'><span
 style='mso-special-format:bullet;position:absolute;left:-3.17%'>&#8226;</span></span><span
 lang=EN-US style='font-family:"Times New Roman";mso-ascii-font-family:"Times New Roman";
 font-size:64%;mso-fareast-language:JA'>Multiplex PCR was performed against 100
 genomic regions simultaneously &#13;</span></span></div>
 <div class=B2 style='mso-line-spacing:"90 20 0";position:absolute;top:48.6%;
 left:16.29%;width:76.96%;height:3.3%'><span style='mso-special-format:nobullet;
 display:none'>&#8226;</span><span lang=EN-US style='font-family:"Times New Roman";
 mso-ascii-font-family:"Times New Roman";font-size:64%;mso-fareast-language:
 JA'><span style='mso-tab-count:1;width:3.15%'> </span>in a single reaction
 tube.&#13;</span></div>
 <div class=B2 style='mso-line-spacing:"90 20 0";position:absolute;top:52.41%;
 left:16.29%;width:76.96%;height:3.3%'><span style='position:absolute;
 top:0%;left:3.16%;width:96.83%'><span style='font-size:64%'><span
 style='mso-special-format:bullet;position:absolute;left:-3.26%'>&#8226;</span></span><span
 lang=EN-US style='font-family:"Times New Roman";mso-ascii-font-family:"Times New Roman";
 font-size:64%;mso-fareast-language:JA'>Each amplified fragment contained at
 least one SNP.&#13;</span></span></div>
 <div class=B2 style='mso-line-spacing:"90 20 0"'><span style='position:absolute;
 top:56.23%;left:18.72%;width:79.02%'><span style='font-size:64%'><span
 style='mso-special-format:bullet;position:absolute;left:-3.08%'>&#8226;</span></span><span
 lang=EN-US style='font-family:"Times New Roman";mso-ascii-font-family:"Times New Roman";
 font-size:64%;mso-fareast-language:JA'>A specialized 384-well card system was
 adopted for the ultra high-throughput </span></span><span style='position:
 absolute;top:59.28%;left:18.72%;width:74.53%'><span lang=EN-US
 style='font-family:"Times New Roman";mso-ascii-font-family:"Times New Roman";
 font-size:64%;mso-fareast-language:JA'>genotyping system.&#13;</span></span></div>
 <div class=B2 style='mso-line-spacing:"90 20 0";position:absolute;top:63.1%;
 left:16.29%;width:76.96%;height:3.3%'><span style='position:absolute;
 top:0%;left:3.16%;width:96.83%'><span style='font-size:64%'><span
 style='mso-special-format:bullet;position:absolute;left:-3.26%'>&#8226;</span></span><span
 lang=EN-US style='font-family:"Times New Roman";mso-ascii-font-family:"Times New Roman";
 font-size:64%;mso-fareast-language:JA'>Very small amount of genomic DNA was
 required for genotyping:&#13;</span></span></div>
 <div class=B3 style='mso-line-spacing:"90 20 0";position:absolute;top:66.92%;
 left:21.16%;width:72.09%;height:3.05%'><span style='position:absolute;
 top:0%;left:3.37%;width:96.62%'><span style='font-size:70%'><span
 style='mso-special-format:bullet;position:absolute;left:-3.49%'>&#8211;</span></span><span
 lang=EN-US style='font-family:"Times New Roman";mso-ascii-font-family:"Times New Roman";
 font-size:70%;mso-fareast-language:JA'>40 ng of genomic DNA was used as a
 template for each multiplex PCR,&#13;</span></span></div>
 <div class=B3 style='mso-line-spacing:"90 20 0";position:absolute;top:70.22%;
 left:21.16%;width:72.09%;height:3.05%'><span style='mso-special-format:nobullet;
 display:none'>&#8211;</span><span lang=EN-US style='font-family:"Times New Roman";
 mso-ascii-font-family:"Times New Roman";font-size:70%;mso-fareast-language:
 JA'><span style='mso-tab-count:1;width:3.36%'> </span> meaning only 0.4 ng of
 genomic DNA per genotyping of one SNP.&#13;</span></div>
 <div class=B1 style='mso-line-spacing:"90 20 0";position:absolute;top:74.04%;
 left:11.42%;width:81.83%;height:3.56%'><span style='position:absolute;
 top:0%;left:3.66%;width:96.1%'><span style='font-size:66%;visibility:hidden'><span
 style='mso-special-format:bullet;position:absolute;left:-3.8%'>&#8211;</span></span><span
 lang=EN-US style='font-size:66%;mso-fareast-language:JA;display:none'>&#13;</span></span></div>
 <div class=B2 style='mso-line-spacing:"90 20 0"'><span style='position:absolute;
 top:78.11%;left:16.29%;width:77.15%'><span style='mso-special-format:nobullet;
 display:none'>&#8226;</span><span lang=EN-US style='font-family:"Times New Roman";
 mso-ascii-font-family:"Times New Roman";font-size:64%;mso-fareast-language:
 JA'>A high-throughput SNP typing system for genome-wide association
 studies.<br>
 </span></span><span style='position:absolute;top:81.17%;
 left:18.72%;width:74.53%'><span lang=EN-US style='font-family:"Times New Roman";
 mso-ascii-font-family:"Times New Roman";font-size:64%;mso-fareast-language:
 JA'>J Hum Genet. 2001;46(8):471-7.<br>
 </span></span><span style='position:
 absolute;top:84.22%;left:18.72%;width:74.53%'><span lang=EN-US
 style='font-family:"Times New Roman";mso-ascii-font-family:"Times New Roman";
 font-size:64%;mso-fareast-language:JA'><p:onmouseclick hyperlinktype="url"
  href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11501945&amp;dopt=Abstract"/><a
 href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11501945&amp;dopt=Abstract"
 target="_parent" onclick="window.event.cancelBubble=true;">Ohnishi Y, Tanaka
 T, Ozaki K, Yamada R, Suzuki H, Nakamura Y</a></span><span lang=EN-US
 style='font-family:"Times New Roman";mso-ascii-font-family:"Times New Roman";
 font-size:57%;mso-fareast-language:JA'><p:onmouseclick hyperlinktype="url"
  href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11501945&amp;dopt=Abstract"/><a
 href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;list_uids=11501945&amp;dopt=Abstract"
 target="_parent" onclick="window.event.cancelBubble=true;">.</a></span><span
 lang=EN-US style='font-family:Arial;mso-ascii-font-family:Arial;mso-fareast-font-family:
 Arial;font-size:57%;mso-fareast-language:JA;mso-special-format:lastCR;
 display:none'>&#13;</span></span></div>
 </div>
 <v:rect id="_x0000_s9221" style='position:absolute;left:44.875pt;top:51.75pt;
  width:765.125pt;height:524.25pt;mso-wrap-style:none;v-text-anchor:middle'
  filled="f" fillcolor="#0c9 [4]" strokecolor="silver" strokeweight=".5pt">
  <v:fill color2="white [0]"/>
  <v:stroke dashstyle="1 1"/>
  <v:shadow color="gray [2]"/>
 </v:rect><![if !vml]><img border=0 v:shapes="_x0000_s9221"
 src="slide0007_image048.gif" style='position:absolute;top:8.14%;left:5.24%;
 width:89.88%;height:83.71%'><![endif]></p:slide></div>

</body>

</html>
