<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:p="urn:schemas-microsoft-com:office:powerpoint"
xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta http-equiv=Content-Type content="text/html; charset=shift_jis">
<meta name=ProgId content=PowerPoint.Slide>
<meta name=Generator content="Microsoft PowerPoint 9">
<link id=Main-File rel=Main-File href="../3-3-2-4-4.htm">
<link rel=Preview href=preview.wmf>
<!--[if !mso]>
<style>
v\:* {behavior:url(#default#VML);}
o\:* {behavior:url(#default#VML);}
p\:* {behavior:url(#default#VML);}
.shape {behavior:url(#default#VML);}
v\:textbox {display:none;}
</style>
<![endif]-->
<title>タイトル</title>
<meta name=Description content="01/11/07: Abstract">
<link rel=Stylesheet href="master03_stylesheet.css">
<![if !ppt]>
<style media=print>
<!--.sld
	{left:0px !important;
	width:6.0in !important;
	height:4.42in !important;
	font-size:107% !important;}
-->
</style>
<script src=script.js></script><script><!--
gId="slide0003.htm"
if( !IsNts() ) Redirect( "PPTSld", gId );
//-->
</script><!--[if vml]><script>g_vml = 1;
</script><![endif]--><script for=window event=onload><!--
if( !IsSldOrNts() ) return;
if( MakeNotesVis() ) return;
LoadSld( gId );
MakeSldVis(0);
//-->
</script><![endif]><o:shapelayout v:ext="edit">
 <o:idmap v:ext="edit" data="5"/>
</o:shapelayout>
</head>

<body lang=JA style='margin:0px;background-color:black'
onclick="DocumentOnClick()" onresize="_RSW()" onkeypress="_KPH()">

<div id=SlideObj class=sld style='position:absolute;top:0px;left:0px;
width:534px;height:393px;font-size:16px;background-color:white;clip:rect(0%, 101%, 101%, 0%);
visibility:hidden'><p:slide coordsize="855,630"
 colors="#FFFFFF,#000000,#808080,#000000,#00CC99,#3333CC,#CCCCFF,#B2B2B2"
 masterhref="master03.xml">
 <p:shaperange href="master03.xml#_x0000_s1025"/><![if !ppt]><p:shaperange
  href="master03.xml#_x0000_s1028"/><![endif]><p:shaperange
  href="master03.xml#_x0000_m1026"/><v:shape id="_x0000_s5122" type="#_x0000_m1026"
  style='position:absolute;left:64.125pt;top:66pt;width:726.75pt;height:50.375pt'>
  <v:fill o:detectmouseclick="f"/>
  <v:stroke o:forcedash="f"/>
  <o:lock v:ext="edit" text="f"/>
  <p:placeholder type="title"/></v:shape><p:shaperange
  href="master03.xml#_x0000_m1027"/><v:shape id="_x0000_s5123" type="#_x0000_m1027"
  style='position:absolute;left:66pt;top:2in;width:10in;height:420pt'>
  <v:fill o:detectmouseclick="f"/>
  <v:stroke o:forcedash="f"/>
  <o:lock v:ext="edit" text="f"/>
  <p:placeholder type="body" position="1"/></v:shape>
 <div v:shape="_x0000_s5122" class=T style='position:absolute;top:10.43%;
 left:8.42%;width:83.14%;height:8.9%;tab-interval:1.1545in'><span lang=EN-US
 style='font-family:"Times New Roman";mso-ascii-font-family:"Times New Roman";
 font-size:98%;mso-fareast-language:JA'>Abstract</span></div>
 <div v:shape="_x0000_s5123" style='tab-interval:.6006in'>
 <div class=B1 style='mso-margin-left-alt:552;mso-text-indent-alt:344'></div>
 <div class=B2 style='mso-margin-left-alt:857;mso-text-indent-alt:691'></div>
 <div class=B style='text-align:justify;mso-line-spacing:"90 20 0";mso-margin-left-alt:
 0;position:absolute;top:23.4%;left:8.61%;width:82.39%;height:18.82%'><span
 style='mso-special-format:nobullet;display:none'>&#8226;</span><span
 style='font-family:Century;mso-ascii-font-family:Century;mso-fareast-font-family:
 "ＭＳ 明朝";font-size:43%'><span style='mso-tab-count:1;width:6.13%'> </span></span><span
 lang=EN-US style='font-family:"Times New Roman";mso-ascii-font-family:"Times New Roman";
 mso-fareast-font-family:"ＭＳ 明朝";font-size:49%;mso-fareast-language:JA'>Linkage
 disequilibrium (LD) mapping, using a dense map of single nucleotide
 polymorphisms (SNPs), has been advocated as the method of choice to locate
 loci of complex genetic diseases. However it has also been suggested that
 genetic population structure could lead to many spurious associations between
 genetic markers and a disease phenotype when LD mapping was performed,
 especially in case of studies using samples from ethnically structured
 populations. &#13;</span></div>
 <div class=B style='text-align:justify;mso-line-spacing:"90 20 0";mso-margin-left-alt:
 0;position:absolute;top:42.74%;left:8.61%;width:82.39%;height:43.51%'><span
 style='mso-special-format:nobullet;display:none'>&#8226;</span><span lang=EN-US
 style='font-family:"Times New Roman";mso-ascii-font-family:"Times New Roman";
 mso-fareast-font-family:"ＭＳ 明朝";font-size:49%;mso-fareast-language:JA'><span
 style='mso-tab-count:1;width:6.13%'> </span>Japanese population has been
 considered to be less structured, and, therefore, more appropriate for LD
 mapping than many other populations in the world, because of its ethnic
 isolation from both historical and geographical standpoints. However no
 analysis on their genetic homogeneity based on assayed genotype data has been
 reported. We evaluated homogeneity of two sub-populations, each of which were
 from one of the two largest Japanese metropolises by analyzing genotype data
 of 188 individuals for 303 unlinked SNPs on autosomal chromosomes. We adopted
 two analytical methods to evaluate structure. One method was to assess
 existence of difference in population structure between two sub-populations,
 by analyzing chi square statistics computed for multiple contingency tables
 constructed for allele distribution of each SNP between sampling
 sub-populations. The other was to infer population structure by model-based
 clustering method using Markov Chain Monte Carlo algorithm. We found that, if
 any, only subtle difference in population structure between two
 sub-populations existed. The possible allele difference between two
 sub-populations seemed small enough not to severely interfere case-control
 association studies for complex genetic traits.</span></div>
 </div>
 <v:rect id="_x0000_s5125" style='position:absolute;left:44.875pt;top:48pt;
  width:765.125pt;height:524.25pt;mso-wrap-style:none;v-text-anchor:middle'
  filled="f" fillcolor="#0c9 [4]" strokecolor="silver" strokeweight=".5pt">
  <v:fill color2="white [0]"/>
  <v:stroke dashstyle="1 1"/>
  <v:shadow color="gray [2]"/>
 </v:rect><![if !vml]><img border=0 v:shapes="_x0000_s5125"
 src="slide0003_image003.gif" style='position:absolute;top:7.63%;left:5.24%;
 width:89.88%;height:83.71%'><![endif]></p:slide></div>

</body>

</html>
